app/viewModels/document/DocumentAnswersViewModel.scala

```diff
@@ -29,28 +31,40 @@ object DocumentAnswersViewModel {
 
   class DocumentAnswersViewModelProvider @Inject() () {
 
-    def apply(userAnswers: UserAnswers, index: Index)(implicit messages: Messages): DocumentAnswersViewModel = {
+    def apply(userAnswers: UserAnswers, index: Index)(implicit messages: Messages, phaseConfig: PhaseConfig): DocumentAnswersViewModel = {
       val helper = new DocumentAnswersHelper(userAnswers, index)
 
-      val section = Section(
-        rows = Seq(
-          helper.attachToAllItems,
-          helper.documentType,
-          helper.previousDocumentType,
-          helper.documentReferenceNumber,
-          helper.typeOfPackageYesNo,
-          helper.packageType,
-          helper.numberOfPackagesYesNo,
-          helper.numberOfPackage,
-          helper.declareQuantityOfGoodsYesNo,
-          helper.metric,
-          helper.quantity,
-          helper.lineItemNumberYesNo,
-          helper.lineItemNumber,
-          helper.additionalInformationYesNo,
-          helper.additionalInformation
-        ).flatten
-      )
+      val rows = Seq(
+        helper.documentType,
+        helper.previousDocumentType,
+        helper.documentReferenceNumber,
+        helper.typeOfPackageYesNo,
+        helper.packageType,
+        helper.numberOfPackagesYesNo,
+        helper.numberOfPackage,
+        helper.declareQuantityOfGoodsYesNo,
+        helper.metric,
+        helper.quantity,
+        helper.lineItemNumberYesNo,
+        helper.lineItemNumber,
+        helper.additionalInformationYesNo,
+        helper.additionalInformation
+      ).flatten
+
+      val postTransitionRows = Seq(helper.attachToAllItems).flatten ++ rows
+
+      val section = {
+        phaseConfig.phase match {
+          case Phase.Transition =>
+            Section(
+              rows = rows
+            )
+          case Phase.PostTransition =>
+            Section(
+              rows = postTransitionRows
+            )
+        }
+      }
```

Comment:
Is this logic needed? We just present whatever they've answered. The nav will handle which questions they answer